src/ProductImport.php

```diff
@@ -48,7 +63,8 @@ public static function createFromAdapter(Adapter $adapter)
         return new self(
             MagentoEavInfo::createFromAdapter($adapter),
             new Sql($adapter),
-            TableResolverFactory::createFromAdapter($adapter)
+            TableResolverFactory::createFromAdapter($adapter),
+            ProductInfoFactory::createFromAdapter($adapter)->create()
```

Comment:
@IvanChepurnyi I'm not sure if this is the right way to create a ProductInfo class, but this works.

Reply:
No problem with that at all! Next time ping me in MageChat as sometimes GitHub is too noisy with notifications that I miss completely new PR.